README.md

```diff
@@ -60,22 +60,39 @@ A third-party plugin is required to enable syntax coloring. The Che4z basic stac
 	
 ## Copybook Support
 
-The LSP for COBOL extension can retrieve copybooks used in your projects from the mainframe and download them locally. You can open copybooks in your IDE and make use of the copybook support features of the extension.
+The COBOL Language Support extension can retrieve copybooks used in your projects from the mainframe and download them locally. You can also store copybooks locally in folders in your workspace. Copybook support features of the extension are enabled whenever a copybook called in source code is successfully located in a specified mainframe data set or local folder.
 
-#### Retrieving Copybooks
+Ensure that copybook names are specified in your source code in upper case, for example `COPY BOOK1` and not `COPY book1`.
```

Comment:
Lower case copybooks' names won't be resolved and stay marked as an error.

Reply:
done in latest commit